core/imt/inc/ROOT/TPoolManager.hxx

```diff
@@ -61,7 +68,7 @@ namespace ROOT {
          TPoolManager(UInt_t nThreads = 0);
          static UInt_t fgPoolSize;
          bool mustDelete = true;
-         tbb::task_scheduler_init *fSched = nullptr;
+         std::unique_ptr<RArenaPtr> fArena;
```

Comment:
Could change the order of the members. Otherwise, you introduce padding after the bool.

Reply:
TIL